launcher_scripts/conf/training/gpt3/175b_performance.yaml

```diff
@@ -0,0 +1,233 @@
+run:
+  name: gpt3_175b
+  results_dir: ${base_results_dir}/${.name}
+  time_limit: "26-00:00:00"
+  dependency: "singleton"
```

Comment:
Please add a comment at the top explaining the reason for this config and the risk with usin git.